docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -151,7 +151,7 @@ The following manifest describes a single-instance WordPress Deployment and Serv
 
    You should see the WordPress set up page similar to the following screenshot.
 
-   ![wordpress-init](https://github.com/kubernetes/examples/blob/master/mysql-wordpress-pd/WordPress.png)
+   [wordpress-init](https://raw.githubusercontent.com/kubernetes/examples/master/mysql-wordpress-pd/WordPress.png)
```

Comment:
Should still have the `!` before it to display the image correct?

Reply:
The original tutorial has `!` before it, but the link is hidden: https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/

The preview after this PR's change shows that the link works after removing `!`: https://deploy-preview-5424--kubernetes-io-master-staging.netlify.com/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/

Reply:
It wasn't meant to be a link, it's an image inlined. https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images

Reply:
I see. I was confused by the original tutorial does not show up any image. Should we have a link to the image instead?

Reply:
Yup, it works now.